README.md

```diff
@@ -48,8 +48,8 @@ Ansible Best Practices
 ** When a single large role is doing many complicated tasks and cannot easily be broken into multiple roles,
    but the process proceeds in multiple related stages
 * Avoid calling the `package` module iteratively with the `{{ item }}` argument, as this is impressively
-  more slow than calling it with the line `name: "{{ foo_packages | join(',') }}"`. The same can go for
-  many other packages that can be given an entire list of items all at once.
+  more slow than calling it with the line `name: "{{ foo_packages }}"`.  The same can go for many other
```

Comment:
Why this change? I always pass in the `| join(',')` filter for the package module. In my experience, it fails hardmode when I try to pass in an actual array.

Reply:
It shouldn't fail...here's a working example from the playbook I've been hacking on recently:
group vars:
```yaml
rpm_dependencies:
  - atomic-openshift-clients
  - docker
  - git
  - apb
```
The task:
```yaml
    - name: Install dependencies
      package:
        name: "{{ rpm_dependencies }}"
        state: present
```

Mainly, though, the reason for the change is that it's the documented way to do it. Joining on commas is listed as an option for yum and dnf, but for all the other package managers I checked they only appear to support a single package spec or a list of them with no mention of a comma-separated list.

Both yum and dnf say "To operate on several packages this can accept a comma separated list of packages or (as of 2.0) a list of packages" for the `name` key (I am brazenly assuming that we're not attempting to work anywhere pre-2.0). Apt specifically says that it takes a list for `name`: "A list of package names, like foo, or package specifier with version, like foo=1.0. Name wildcards (fnmatch) like apt* and version wildcards like foo=1.0* are also supported", though I assume it would probably also work fine with a single package string or comma-separated list despite those behaviors being undocumented. pacman says "Name or list of names of the packages to install, upgrade, or remove" and zypper says "Package name name or package specifier or a list of either."

I did some local testing with pacman, and all these synaxes were supported and did what you'd expect:
```yaml
- package:
    name: cowsay
- package:
    name: linux,cowsay
- package:
    name:
      - linux
      - cowsay
```